dashboard/app/dsl/script_dsl.rb

```diff
@@ -192,15 +193,19 @@ def level(name, properties = {})
       if challenge
         @current_scriptlevel[:properties][:challenge] = challenge
       end
+      if named
+        @current_scriptlevel[:properties][:named] = named
```

Comment:
I think it is going to be better if we set `[:named_level]` here instead, so that we don't have two ways to identify script levels as named.